doc/src/guides/solving/solve-system-of-equations-algebraically.md

```diff
@@ -94,7 +103,10 @@ could be satisfied if $z=0$. Note that {func}`~.solve` will not assume that
 $z=0$ even though that is the only value of $z$ that makes the system of
 equations consistent because $z$ is a parameter rather than an unknown. That is,
 {func}`~.solve` does not treat $z$ as an unknown because it is not in the list
-of symbols to be solved for `[x, y]`.
+of symbols to be solved for `[x, y]`. (Note that you define a symbol the same
+way, for example `from sympy.abc import x, y, z`, whether it is an unknown or a
+parameter; whether a symbol is treated as an unknown to be solved for hinges on
+whether that symbol is in the list of unknowns passed to `solve`.)
```

Comment:
of symbols specified as unknowns (`[x, y]`) and all such symbols are treated like parameters with arbitrary value. This is a semantic definition and has nothing to do with a symbol's creation (or import from `abc`).